.tool-versions

```diff
@@ -0,0 +1,3 @@
+maven 3.9.1
+nodejs 18.14.2
```

Comment:
node is required ?

Reply:
It depends, if you want to use the prettier IDE extension for formatting when editing code, yes.

If you only format code with the maven plugin, no.

Reply:
As this is a public repo, I think that we don't need to add it to avoid confusion. Maybe you can add this in an upper level.